core/src/test/scala/pureconfig/ConfigConvertChecks.scala

```diff
@@ -78,7 +78,7 @@ trait ConfigConvertChecks { this: FlatSpec with Matchers with GeneratorDrivenPro
    */
   def checkFailure[T, E <: ConfigReaderFailure](values: ConfigValue*)(implicit cr: ConfigConvert[T], tag: ClassTag[T], eTag: ClassTag[E]): Unit =
     for (value <- values) {
-      it should s"fail when it tries to read a value of type ${tag.runtimeClass.getSimpleName} " +
+      it should s"fail when it tries to read a value of type ${tag.runtimeClass.getCanonicalName} " +
```

Comment:
I'm ok with it, in this case I suppose it helps to distinguish a Scala Duration from a Java one.